### ISSUE TITLE
Add Site on top of Location to manage multiple homes

### ISSUE THREAD

**Issue by drtech981** [hay-kot/homebox#115]
Would be great if the developers could add another hierarchy "SITES" above the Location Level !

Kudos to this great project !

**Comment by proffalken** [comment]
I just see additional sites as other locations tbh.

So `My House`, `Storage Unit 01` and `Workshop` are all top level "locations", then within these I have sub-locations such as `living room`, `Shelving Rack 01`, `Tool Rack 02`.  

These sub-locations then may or may not have additional sub-locations such as `bookshelf by door`, `TV Unit`, or `Drawer 03`, giving me a hierarchy similar to this:

```
|- My House
|        |- Living Room
|                |- Bookshelf by Door
|                |- TV Unit
|
|- Storage Unit
|        |- Shelving Rack 01
|                 |- Drawer 03
|
|- Workshop
        |- Tool rack 02
```

I guess it's just about naming things and how much clarity you want?

**Comment by hay-kot** [comment]
@drtech981, nested locations have made it into the nightly, so I think that that's the approach we're going to take with this one. Given that you can use locations for arbitrary deep nesting I think that should cover most use cases here and doesn't require us to create a duplicate model on the backend that would be a Location with a different name. 

What we might look at in the future is changing the dashboard to only show locations that don't have parents (i.e only top level locations), and then have a separate locations page that lists all of them. I _think_ that that would be the behavior that most people would want, but I'll need to get some more feedback from users to be sure.

**Comment by proffalken** [comment]
> What we might look at in the future is changing the dashboard to only show locations that don't have parents (i.e only top level locations), and then have a separate locations page that lists all of them

I was thinking about logging an issue specifically for this earlier!

I'm up to about 30 locations now, 5 levels deep in some places, and having them all displayed on the front page is getting messy! 🤣

**Comment by hay-kot** [comment]
> I was thinking about logging an issue specifically for this earlier!

Fixed in #127!

**Comment by hay-kot** [comment]
Going to close this as `won't fix` since we're solving this problem in a different sort of way. Thanks for your input!
